index.js

```diff
@@ -86,6 +87,56 @@ Lingo.prototype.fetchSection = function(id, version, page = 1, limit = 50) {
   });
 };
 
+/**
+ * Utility function to fetch all items in a section, automatically paging if needed.
+ *
+ * The API limits fetches to 200. This function recursively calls fetchSection until all items have been retrieved.
+ * To page manually, use `fetchSection`
+ *
+ * @param {uuid} id the section uuid
+ * @param {integer} version the version number of the section to fetch
+ * @param {integer} limit The max number of items to fetch
+ * @returns {Promise} A promise resolving the section and the items matching the page/limit
+ */
+Lingo.prototype.fetchAllItemsInSection = function (id, version = 0) {
+  return new Promise((resolve, reject) => {
+    let page = 1;
+    const limit = 200 // API Enforces <= 200
+    let results = [];
+
+    const self = this;
+    function fetch() {
+      self
+        .fetchSection(id, version, page, limit)
+        .then(section => {
+          const items = section.items
+          results = [...results, ...items]
+          if (items.length < limit) {
```

Comment:
I can't remember what the API does if you request an empty page. If it returns an error this could be a problem, but I think maybe it just responds with an empty array?

Reply:
98% sure it's just an empty array.